### ISSUE TITLE
admit .rmd extension

### ISSUE THREAD

**Issue by corybrunson** [hadley/pkgdown#440]
I have a (bad?) habit of naming my Rmarkdown files with the extension `.rmd` rather than `.Rmd`, so that **pkgdown** throws a cryptic error when building articles from my vignettes:
```sh
# Error: In '_pkgdown.yml', topic must be a valid R expression.
```
I solved the problem locally by an adjustment analogous to #99, though only for vignettes (in `link_article_index.R` and `package.r`) and not for index pages. I'm glad to fork and submit a pull request on a temp branch, if that's the proper etiquette. Thanks for any help or guidance!

**Comment by hadley** [comment]
A PR would be great!

**Comment by corybrunson** [comment]
Addressed by #441.

**Comment by hadley** [comment]
In the future, if you include `Fixes #440` in a commit message, the issue will get closed automatically when the PR is merged.
